controllers/dashboard.go

```diff
@@ -46,9 +46,17 @@ func (c *DashboardController) actionPolicyMapping() {
 func (c *DashboardController) Index() {
 	web.ReadFromRequest(&c.Controller)
 
+	var keyword string
+	err := c.Ctx.Input.Bind(&keyword, "keyword")
```

Comment:
It can be done also with `GetString`, ex: `keyword  := c.GetString("keyword")`. Current one also ok to me.

Reply:
It looks nicer 👍